prep-javascript-objects/main.js

```diff
@@ -0,0 +1,23 @@
+
+var person = {
+  firstName: 'Amir',
+  lastName: 'Alamin',
+  hobby: 'working out'
+};
+console.log(person);
+
+person.job = 'insurance';
+console.log("the person's current job is:", person.job);
+
+person.previousjob = 'retail';
+console.log("the person's previous job is:", person.previousjob);
+
+person.complete = {
+  firstName: 'Amir',
+  lastName: 'Alamin',
+  hobby: 'working out'
+};
+console.log(person);
+
+var fullname = "the person's name is: Amir Alamin";
+console.log(fullname);
```

Comment:
Hi Amir, you can use the `firstName` and `lastName` properties of the `person` object to create the full name.

```js
var fullName = person.firstName + ' ' + person.lastName;
```